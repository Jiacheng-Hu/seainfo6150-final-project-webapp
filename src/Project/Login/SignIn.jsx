import React from "react";

    const SignIn = () => {
    return (
        <div className="signin">
            <h2 className="text-center">Please sign in</h2>
            <div className="margin-top">Username</div>
            <input type="text" name="username" id="username" required className="form-control"/>
            <div className="margin-top">Password</div>
            <input type="password" name="password" id="password" required  className="form-control"/><br/>
            <input type="submit" value="submit" className="btn btn-lg btn-primary btn-block margin-top"/>
            <input type="reset" value="reset" className="btn btn-lg btn-default btn-block margin-top"/>
          
        </div>
    );
    
};

export default SignIn;