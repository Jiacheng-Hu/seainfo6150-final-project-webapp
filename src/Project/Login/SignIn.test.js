import React from "react";
import SignIn from "./SignIn";


describe("SignIn tests", () => {
    it("renders correctly", () => {
        const {container} = render(<SignIn/>);
        expect(container).toMatchSnapshot();
    });
});