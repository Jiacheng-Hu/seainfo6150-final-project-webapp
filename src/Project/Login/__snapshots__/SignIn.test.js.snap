// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn tests renders correctly 1`] = `
<div>
  <div
    class="signin"
  >
    <h2
      class="text-center"
    >
      Please sign in
    </h2>
    <div
      class="margin-top"
    >
      Username
    </div>
    <input
      class="form-control"
      id="username"
      name="username"
      required=""
      type="text"
    />
    <div
      class="margin-top"
    >
      Password
    </div>
    <input
      class="form-control"
      id="password"
      name="password"
      required=""
      type="password"
    />
    <br />
    <input
      class="btn btn-lg btn-primary btn-block margin-top"
      type="submit"
      value="submit"
    />
    <input
      class="btn btn-lg btn-default btn-block margin-top"
      type="reset"
      value="reset"
    />
  </div>
</div>
`;
